dot/state/service_integration_test.go

```diff
@@ -176,9 +175,9 @@ func TestService_StorageTriePruning(t *testing.T) {
 	config := Config{
```

Comment:
The test `TestService_StorageTriePruning` which was modified in #3063 should now be unskipped.